diff --git a/node_modules/@trivago/prettier-plugin-sort-imports/lib/src/utils/extract-ast-nodes.js b/node_modules/@trivago/prettier-plugin-sort-imports/lib/src/utils/extract-ast-nodes.js
index 819a91c..54c1bb7 100644
--- a/node_modules/@trivago/prettier-plugin-sort-imports/lib/src/utils/extract-ast-nodes.js
+++ b/node_modules/@trivago/prettier-plugin-sort-imports/lib/src/utils/extract-ast-nodes.js
@@ -10,11 +10,17 @@ function extractASTNodes(ast) {
     var importNodes = [];
     var directives = [];
     traverse_1.default(ast, {
-        Directive: function (_a) {
-            var node = _a.node;
-            directives.push(node);
-            // Trailing comments probably shouldn't be attached to the directive
-            node.trailingComments = null;
+        Directive: function (path) {
+            // Only capture directives if they are at the top scope of the source
+            // and their previous siblings are all directives
+            if (path.parent.type === 'Program' &&
+                path.getAllPrevSiblings().every(function (s) {
+                    return s.type === 'Directive';
+                })) {
+                directives.push(path.node);
+                // Trailing comments probably shouldn't be attached to the directive
+                path.node.trailingComments = null;
+            }
         },
         ImportDeclaration: function (path) {
             var tsModuleParent = path.findParent(function (p) {